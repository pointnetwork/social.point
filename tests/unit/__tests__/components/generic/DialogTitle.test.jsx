import React from "react";
import renderer from 'react-test-renderer';

import DialogTitle from '../../../../../src/components/generic/DialogTitle';

describe("DialogTitle", () => {

    it('should render', () => {
        const component = renderer.create(
            <DialogTitle classes={{}} children={<div></div>} onClose={jest.fn()} other={{}}></DialogTitle>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});