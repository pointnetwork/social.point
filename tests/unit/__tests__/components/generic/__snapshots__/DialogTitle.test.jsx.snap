// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogTitle should render 1`] = `
<div
  className="MuiDialogTitle-root Component-root-1"
  other={Object {}}
>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    <div />
  </h6>
  <button
    aria-label="close"
    className="MuiButtonBase-root MuiIconButton-root Component-closeButton-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[MockFunction]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiIconButton-label"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
        />
      </svg>
    </span>
  </button>
</div>
`;
